### PR TITLE
DM-762 [Doc]Review lpwan-custom-codec and correct the Readme

### PR DESCRIPTION
**Pull request by ashhSAG93**

### CODE REVIEW

**Review by miryalarahul — DISMISSED** [review]

**Review by atulpandasag — DISMISSED** [review]

File: microservice/lpwan-custom-codec/readme.md [hunk 1]
```diff
@@ -280,8 +200,12 @@ The fragments above are used as follows:
 
 ```json
 {
-    "encodedCommand": "9F5000",
-    "fport": 20
+  "encodedCommand": "9F000000",
+  "properties": {
+    "fport": "100"
+  },
+  "message": "Successfully Encoded the payload",
```
Comment (rssag) [inline microservice/lpwan-custom-codec/readme.md]: not sure if this is something we can even change in the scope of this PR, but i think it should be "Encoded" -> "encoded" (lower case).

File: microservice/lpwan-custom-codec/readme.md [hunk 2]
```diff
@@ -512,14 +415,13 @@ public class LansitecEncoder implements EncoderService {
...
 ```
 
-## Developing microservice without lpwan-custom-codec
-
-If the user does not use the lpwan-custom-codec, then the custom microservice developed by the user must adhere to the pre-requisite task performed by the lpwan-custom-codec i.e must create the device types and pre-defined commands to use the custom codec feature seamlessly. 
-
 ## Deploying the example codec microservice
 
-In order to build and deploy the sample codec microservice, follow the [Microservice SDK guide](http://www.cumulocity.com/guides/microservice-sdk/java/).
+1. Clone https://github.com/SoftwareAG/cumulocity-examples.git repository.
```
Comment (rssag) [inline microservice/lpwan-custom-codec/readme.md]:
```suggestion
1. Clone the https://github.com/SoftwareAG/cumulocity-examples.git repository.
```

File: microservice/lpwan-custom-codec/readme.md [hunk 3]
```diff
@@ -512,14 +415,13 @@ public class LansitecEncoder implements EncoderService {
...
+2. Build the microservice using `mvn clean install`. This creates a ZIP file of the lanitec codec microservice.
+3. Deploy the microservice by uploading the ZIP file using the Cumulocity IoT Administration UI.
+4. Open the device management application. Under **Device protocols**, you should now see the device types with type "lpwan" created by the lansitec codec microservice.
```
Comment (rssag) [inline microservice/lpwan-custom-codec/readme.md]:
```suggestion
4. Open the Device Management application. Under **Device protocols**, you can now see the device types with type "lpwan" created by the lansitec codec microservice.
```

File: microservice/lpwan-custom-codec/readme.md [hunk 4]
```diff
@@ -512,14 +415,13 @@ public class LansitecEncoder implements EncoderService {
...
 
-First, clone this repository. Next, build the microservice using `mvn clean install`. The build will create a zip file of the codec microservice.
+## Developing microservice without lpwan-custom-codec
```
Comment (rssag) [inline microservice/lpwan-custom-codec/readme.md]: "microservice" -> "a microservice" or "microservices"

File: microservice/lpwan-custom-codec/readme.md [hunk 5]
```diff
@@ -512,14 +415,13 @@ public class LansitecEncoder implements EncoderService {
...
 
-In the next step, deploy the microservice using the Cumulocity IoT UI. Once the decoder microservice has been deployed, it can take couple of minutes for the Cumulocity platform to discover the new decoder. Then, open the device management application. Under device protocols, you should now see the device types with type as 'lpwan' created by the custom codec microservice. Map one of these device types to the LPWAN device.
+If the user does not use the lpwan-custom-codec, then the custom microservice developed by the user must adhere to the pre-requisite task performed by the lpwan-custom-codec i.e must create the device types and pre-defined commands to use the custom codec feature seamlessly.
```
Comment (rssag) [inline microservice/lpwan-custom-codec/readme.md]:
```suggestion
If the user does not use the lpwan-custom-codec, then the custom microservice developed by the user must adhere to the prerequisite task performed by the lpwan-custom-codec, that is, the user must create the device types and predefined commands to use the custom codec feature seamlessly. 
```

File: microservice/lpwan-custom-codec/readme.md (hunk 1 above)
Comment (ashhSAG93) [reply microservice/lpwan-custom-codec/readme.md:207]: @rssag this message is coming from the example microservice lora-codec-lansitec. Hence the case is coming in this manner. We could correct it in a subsequent PR.

**Review by miryalarahul — DISMISSED** [review]

**Review by rssag — APPROVED** [review]

**Review by miryalarahul — APPROVED** [review]

**Pull request merged**